src/main/java/com/example/shopSpring/model/CardData.java

```diff
@@ -0,0 +1,29 @@
+package com.example.shopSpring.model;
+
+import jakarta.persistence.Entity;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "cardData")
+@Getter
+@Setter
+public class CardData {
+
+    @Id
+    private Integer id;
+
+    private String cardNum;
```

Comment:
Every data field should have @Column annotation right? I dont remember it but is is present in this example: https://github.com/spring-projects/spring-petclinic/blob/main/src/main/java/org/springframework/samples/petclinic/model/Person.java

Reply:
done